sdks/java/extensions/sql/jdbc/src/main/java/org/apache/beam/sdk/extensions/sql/jdbc/BeamSqlLine.java

```diff
@@ -19,41 +19,57 @@
 
 import static org.apache.beam.sdk.extensions.sql.impl.JdbcDriver.CONNECT_STRING_PREFIX;
 
-import com.google.common.collect.ImmutableList;
 import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import javax.annotation.Nullable;
 import sqlline.SqlLine;
+import sqlline.SqlLine.Status;
 
 /** {@link BeamSqlLine} provides default arguments to SqlLine. */
 public class BeamSqlLine {
 
   private static final String NICKNAME = "BeamSQL";
 
   public static void main(String[] args) throws IOException {
+    runSqlLine(checkConnectionArgs(args), null, null, null);
```

Comment:
i think it's better to call `checkConnectionArgs()` within `runSqlLine()`, it's basically the only thing that happens in this class, we probably should exercise it